Pinta.Core/Managers/WorkspaceManager.cs

```diff
@@ -219,35 +219,21 @@ public double Scale {
 	public ReadOnlyCollection<Document> OpenDocuments { get; }
 	public bool HasOpenDocuments => open_documents.Count > 0;
 
-	public Document CreateAndActivateDocument (
-		ActionManager actions,
-		Gio.File? file,
-		string? file_type,
-		Size size)
+	public void AttachDocument (
```

Comment:
I'm not sure how much I like the `Attach` terminology, since that makes me think there should be a corresponding `Detach` somewhere. But I don't think it's important for documents to have the notion of a special attached vs detached state - although they might be created standalone just so that file import code doesn't need to be aware of the workspace etc, after that they are always added into a workspace until the workspace closes the document

I think something like `AddDocument()` would be more clear, or `ActivateDocument()` to follow the original naming

Reply:
The naming of the methods and events is now back to `ActivateDocument`